conf/schema.xml

```diff
@@ -431,6 +442,12 @@
 <!-- The following line is a bit of a performance hog, I would look to remove it if the indexing takes a long time. -->
 <copyField source="*" dest="catch_all_fields_mt"/>
 
+<!-- BPL Compressed Search Facets -->
+<copyField source="mods_titleInfo_title_s" dest="mods_titleInfo_title_sci"/>
```

Comment:
Using *_s fields only returns the first value if there are multiple. Need to copy the *_ms fields to get all expected values.

Reply:
to clarify, looks like we need to copy both the _s fields and the _ms fields

Reply:
Copying both _ms and _s fields for mods_titleInfo_title_* and mods_subject_topic_*